### PR TITLE
Switch email and user name and change submit button to sign up

### PR DESCRIPTION
**Pull request by Kendiya242**

### CODE REVIEW

File: views/pages/registration-form.ejs [hunk 1]
```diff
@@ -58,28 +58,30 @@
...
-                                placeholder="Username" 
-                                autocomplete="username"
-                                required
+                          <!-- Email -->
+                          <input class="input-field"
+                            type="email" 
+                            name="email" 
+                            id="email" 
+                            index: true,
+                            placeholder="mmcfly@cusco.com"
```
Comment (dinaolmelak) [inline views/pages/registration-form.ejs]: Can you change this placeholder to be `Email`?

**Review by dinaolmelak — CHANGES_REQUESTED** [review]
Seems good 👍🏽, I was hoping we could get 2 birds with 1 stone with the following Requested changes

File: views/pages/registration-form.ejs (hunk 1 above)
Comment (Kendiya242) [reply views/pages/registration-form.ejs]: i totally forget but I am on it

**Review by dinaolmelak — APPROVED** [review]
LGTM

File: views/pages/registration-form.ejs (hunk 1 above)
Comment (dinaolmelak) [reply views/pages/registration-form.ejs]: No Worries  😄

**Pull request merged**